src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/Stylus/Common/StylusPointDescription.cs

```diff
@@ -296,10 +296,8 @@ internal int OriginalPressureIndex
         /// <param name="stylusPointDescription2">stylusPointDescription2</param>
         public static bool AreCompatible(StylusPointDescription stylusPointDescription1, StylusPointDescription stylusPointDescription2)
         {
-            if (stylusPointDescription1 == null || stylusPointDescription2 == null)
-            {
-                throw new ArgumentNullException("stylusPointDescription");
-            }
+            ArgumentNullException.ThrowIfNull(stylusPointDescription1);
+            ArgumentNullException.ThrowIfNull(stylusPointDescription2);
```

Comment:
I deliberately changed the error message here to reflect the actual parameter names